client/src/components/Review Widget/ReviewStars.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import quarterStar from './icons/star-one-quarter.svg'
+import fullStar from './icons/star.svg'
+import halfStar from './icons/star-half.svg'
+import threeQuarterStar from './icons/star-three-quarter.svg'
+
+const ReviewStars = (props) => {
+  const wholeStars = [...Array(Math.floor(props.starRating))]
+  const decimalStars = []
+  const decimalRating = props.starRating % 1
+
+  for (let i = 0; i < wholeStars.length; i++) {
+    wholeStars[i] = <img src={fullStar} />
```

Comment:
I think we might need a key property here for react. Rendered lists in react typically result in an error in the console. Although this error is 'optional', we should probably fix these eventually to avoid console clutter.

Reply:
agreed, i've just never been able to get the key to get rid of the error...:/
 I'll dig a little more into the documentation for this and fix it before my next PR though